core/src/main/java/org/mskcc/oncotree/model/TumorType.java

```diff
@@ -12,36 +29,37 @@
 @javax.annotation.Generated(value = "class io.swagger.codegen.languages.SpringMVCServerCodegen", date = "2016-04-04T17:16:11.368Z")
 public class TumorType {
 
-    private Integer id = null;
     private String code = null;
     private String color = null;
     private String name = null;
-    private MainType mainType = null;
+    private String mainType = null;
     private List<String> NCI = new ArrayList<String>();
```

Comment:
The fields NCI and UMLS make these links to external annotations a top-level part of the model. That is not inherently wrong, but each new mapping to an external ontology may require a change to our model and break the API for external users. (A major revision change). So to avoid that, I think we should consider representing links to external vocabularies in a single top level object ... perhaps a map from vocabulary names {NCI, UMLS, ...} to string values.